x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/execution/search/QlSourceBuilder.java

```diff
@@ -23,6 +23,7 @@
  */
 public class QlSourceBuilder {
     public static final Version SWITCH_TO_FIELDS_API_VERSION = Version.V_7_10_0;
+    public static final Version MISSING_ORDER_IN_COMPOSITE_AGGS_VERSION = Version.V_7_16_0;
```

Comment:
This name makes sense now in this PR, but I'd prolly find it a bit confusing later. Imo it'd help a comment or something like `INTRODUCING_` prefix. Just a minor suggestion.

Reply:
makes sense, I'll add the `INTRODUCING_` prefix 👍